frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -94,6 +96,21 @@
 
   let userModal: UserModal;
 
+  let loadingRepoSize = false;
+  async function updateRepoSize(): Promise<void> {
+    loadingRepoSize = true;
+    await _updateProjectRepoSizeInKb(project.code);
+    loadingRepoSize = false;
+  }
+
+  $: if (project.repoSizeInKb == null) {
+    void updateRepoSize();
```

Comment:
can we move this to run on mount? I'm worried about some bug which causes this to run repeatedly hammering our server.

Reply:
Yeah, on mount is what makes sense. Then it only runs once, and even then only if there's no repo size returned from GQL.